examples/text_summarization/pointer_summarizer/utils/batcher.py

```diff
@@ -0,0 +1,282 @@
+#Most of this file is copied form https://github.com/abisee/pointer-generator/blob/master/batcher.py
```

Comment:
Please make sure the License and author is correct and clear.
And you need to unify the coding style to PEP8